css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  right: 600px;
```

Comment:
You should just make this element `width: 100%` so that `text-align` can do its job 🤓